ui/pages/home/home.component.js

```diff
@@ -415,10 +424,13 @@ export default class Home extends PureComponent {
                     {t('newTokensImportedMessage', [newTokensImported])}
                   </Typography>
                 </Box>
-                <button
-                  className="fas fa-times home__new-tokens-imported-notification-close"
-                  title={t('close')}
+
+                <ButtonIcon
+                  iconName={ICON_NAMES.CLOSE}
+                  size={ICON_SIZES.XS}
+                  ariaLabel={t('close')}
                   onClick={() => setNewTokensImported('')}
+                  className="home__new-tokens-imported-notification-close"
                 />
               </Box>
             }
```

Comment:
Would be good to get some screenshots. The styles associated with this classname are adding a close icon via unicode so it could be showing double X icons right now?

<img width="502" alt="Screenshot 2023-02-21 at 11 04 06 PM" src="https://user-images.githubusercontent.com/8112138/220536617-609ca081-3885-447c-be2c-e146497a6a1b.png">

Reply:
The style you're citing is only in swaps.

Reply:
<img width="354" alt="Screenshot at Feb 22 12-42-41" src="https://user-images.githubusercontent.com/46655/220727429-4f95b608-7b69-4762-a8af-611e40c15a28.png">